re/service.go

```diff
@@ -460,6 +467,37 @@ func (re *re) sendEmail(L *lua.LState) int {
 	return 1
 }
 
+func (re *re) saveSenml(L *lua.LState) int {
+	luaString := L.ToString(1)
+	var message senml.Message
+
+	if err := json.Unmarshal([]byte(luaString), &message); err != nil {
+		return 0
+	}
+
+	payload, err := json.Marshal(message)
+	if err != nil {
+		return 0
+	}
+
+	domainId := L.GetGlobal("domain_id").String()
+
+	ctx := context.Background()
+	m := &messaging.Message{
+		Publisher: message.Publisher,
+		Created:   time.Now().Unix(),
```

Comment:
We need to take time from the message. Please check if any transformation is necessary.